src/engines/L3/element.js

```diff
@@ -205,6 +205,11 @@ const propsTransformer = {
     }
   },
   set effects(v) {
+    for (let i = 0; i < v.length; i++) {
+      if (v[i].props && v[i].props.color) {
```

Comment:
maybe this should be a `'props' in v[i] === true` check (and `'color' in v[i].props` as well). I think that's (slightly) faster